### PR TITLE
fix: Add packages to pkg-lists

### PR DESCRIPTION
**Pull request by rayjh**
Add wget and ntp to package lists. These were not resolved by package
dependency tooling as they are included in the pup kickstart file.

### CODE REVIEW

File: software/pkg-lists-wmla120.yml [hunk 1]
```diff
@@ -545,6 +545,12 @@ yum_pkgs_p8:
     - xorg-x11-xkb-utils-7.7-14.el7.ppc64le
     - yum-utils-1.1.31-50.el7.noarch
     - zlib-devel-1.2.7-18.el7.ppc64le
+# Do not remove the additional packages below. They are included
+# in the pup kickstart and are thus not detected in package dependency
+# resolution
+    - wget
```
Comment (jjalvare) [inline software/pkg-lists-wmla120.yml]: Can you please add these  packages to: pkg-lists-wmla120_x86_64.yml ?

Comment (rayjh) [reply software/pkg-lists-wmla120.yml]: done

Comment (rayjh) [reply software/pkg-lists-wmla120.yml]: verified on p8 node

Comment (jjalvare) [reply software/pkg-lists-wmla120.yml]: verified on x86

**Pull request merged**